### ISSUE TITLE
Odd braille translation that causes RuntimeError (inbuf b'R\x00\x00\x00e\x00')

### ISSUE THREAD

**Issue by Andre9642** [nvaccess/nvda#10370]
### Steps to reproduce:
With braille enabled, revert to saved configuration *at least 2 times*. It's not the only way but it's hard to reproduce it otherwise.

### Actual behavior:
Works but an error is raised sometimes:

<details><summary>Traceback</summary>

```
Traceback (most recent call last):
  File "eventHandler.pyc", line 155, in executeEvent
  File "eventHandler.pyc", line 92, in __init__
  File "eventHandler.pyc", line 100, in next
  File "NVDAObjects\__init__.pyc", line 1037, in event_gainFocus
  File "braille.pyc", line 1834, in handleGainFocus
  File "braille.pyc", line 1839, in _doNewObject
  File "braille.pyc", line 1520, in getFocusRegions
  File "braille.pyc", line 619, in update
  File "braille.pyc", line 429, in update
  File "louisHelper.pyc", line 65, in translate
  File "louis\__init__.pyc", line 184, in translate
RuntimeError: Can't translate: tables ['louis\\tables\\fr-bfu-comp8.utb', 'braille-patterns.cti'], inbuf b'R\x00\x00\x00e\x00\x00\x00c\x00\x00\x00h\x00\x00\x00a\x00\x00\x00r\x00\x00\x00g\x00\x00\x00e\x00\x00\x00r\x00\x00\x00 \x00\x00\x00d\x00\x00\x00l\x00\x00\x00g\x00\x00\x00 \x00\x00\x00"\x00\x00\x00C\x00\x00\x00:\x00\x00\x00\\\x00\x00\x00U\x00\x00\x00s\x00\x00\x00e\x00\x00\x00r\x00\x00\x00s\x00\x00\x00\\\x00\x00\x00a\x00\x00\x00n\x00\x00\x00d\x00\x00\x00r\x00\x00\x00e\x00\x00\x00\\\x00\x00\x00A\x00\x00\x00p\x00\x00\x00p\x00\x00\x00D\x00\x00\x00a\x00\x00\x00t\x00\x00\x00a\x00\x00\x00\\\x00\x00\x00R\x00\x00\x00o\x00\x00\x00a\x00\x00\x00m\x00\x00\x00i\x00\x00\x00n\x00\x00\x00g\x00\x00\x00\\\x00\x00\x00n\x00\x00\x00v\x00\x00\x00d\x00\x00\x00a\x00\x00\x00\\\x00\x00\x00n\x00\x00\x00v\x00\x00\x00d\x00\x00\x00a\x00\x00\x00.\x00\x00\x00i\x00\x00\x00n\x00\x00\x00i\x00\x00\x00"\x00\x00\x00\n\x00\x00\x00\n\x00\x00\x00C\x00\x00\x00e\x00\x00\x00 \x00\x00\x00f\x00\x00\x00i\x00\x00\x00c\x00\x00\x00h\x00\x00\x00i\x00\x00\x00e\x00\x00\x00r\x00\x00\x00 \x00\x00\x00a\x00\x00\x00 \x00\x00\x00\xe9\x00\x00\x00t\x00\x00\x00\xe9\x00\x00\x00 \x00\x00\x00m\x00\x00\x00o\x00\x00\x00d\x00\x00\x00i\x00\x00\x00f\x00\x00\x00i\x00\x00\x00\xe9\x00\x00\x00 \x00\x00\x00p\x00\x00\x00a\x00\x00\x00r\x00\x00\x00 \x00\x00\x00u\x00\x00\x00n\x00\x00\x00 \x00\x00\x00a\x00\x00\x00u\x00\x00\x00t\x00\x00\x00r\x00\x00\x00e\x00\x00\x00 \x00\x00\x00p\x00\x00\x00r\x00\x00\x00o\x00\x00\x00g\x00\x00\x00r\x00\x00\x00a\x00\x00\x00m\x00\x00\x00m\x00\x00\x00e\x00\x00\x00.\x00\x00\x00\n\x00\x00\x00V\x00\x00\x00o\x00\x00\x00u\x00\x00\x00l\x00\x00\x00e\x00\x00\x00z\x00\x00\x00-\x00\x00\x00v\x00\x00\x00o\x00\x00\x00u\x00\x00\x00s\x00\x00\x00 \x00\x00\x00l\x00\x00\x00e\x00\x00\x00 \x00\x00\x00r\x00\x00\x00e\x00\x00\x00c\x00\x00\x00h\x00\x00\x00a\x00\x00\x00r\x00\x00\x00g\x00\x00\x00e\x00\x00\x00r\x00\x00\x00 \x00\x00\x00e\x00\x00\x00t\x00\x00\x00 \x00\x00\x00p\x00\x00\x00e\x00\x00\x00r\x00\x00\x00d\x00\x00\x00r\x00\x00\x00e\x00\x00\x00 \x00\x00\x00l\x00\x00\x00e\x00\x00\x00s\x00\x00\x00 \x00\x00\x00m\x00\x00\x00o\x00\x00\x00d\x00\x00\x00i\x00\x00\x00f\x00\x00\x00i\x00\x00\x00c\x00\x00\x00a\x00\x00\x00t\x00\x00\x00i\x00\x00\x00o\x00\x00\x00n\x00\x00\x00s\x00\x00\x00 \x00\x00\x00e\x00\x00\x00f\x00\x00\x00f\x00\x00\x00e\x00\x00\x00c\x00\x00\x00t\x00\x00\x00u\x00\x00\x00\xe9\x00\x00\x00e\x00\x00\x00s\x00\x00\x00 \x00\x00\x00d\x00\x00\x00a\x00\x00\x00n\x00\x00\x00s\x00\x00\x00 \x00\x00\x00N\x00\x00\x00o\x00\x00\x00t\x00\x00\x00e\x00\x00\x00p\x00\x00\x00a\x00\x00\x00d\x00\x00\x00+\x00\x00\x00+\x00\x00\x00 \x00\x00\x00?\x00\x00\x00', typeform None, cursorPos c_long(0), mode 4
```
</details>

This error is annoying for BrailleExtender because if any errors produce on braille modules, some features of add-on are automatically disabled For security purposes.

### Expected behavior:
No error

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Alpha

#### Windows version:
10

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
N/A

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No problem with 2019.2 and 2018.2.

**Comment by lukaszgo1** [comment]
cc @leonardder

**Comment by Andre9642** [comment]
Actually, the real question is: where these '\x00' come? Indeed if we remove all '\x00' in the string, we get:

> Recharger dlg "C:\\Users\\andre\\AppData\\Roaming\\nvda\\nvda.ini"\n\nCe fichier a été modifié par un autre programme.\nVoulez-vous le recharger et perdre les modifications effectuées dans Notepad++ ?

((In english: Reload dlg "C:\\Users\\andre\\AppData\\Roaming\\nvda\\nvda.ini"\n\nThis file has been modified by another program.
Do you want to reload it?))

**Comment by lukaszgo1** [comment]
Can you reproduce this with build from PR #10275? If you cannot it might make sense to upgrade to Liblouis 3.11 for 2019.3 after all.

**Comment by leonardder** [comment]
> Actually, the real question is: where these '\x00' come?

These are normal, they're just null bytes. What you're seeing is a raw string in utf-32/ UCS4.

```
>>> "hello".encode("utf_32_le")
b'h\x00\x00\x00e\x00\x00\x00l\x00\x00\x00l\x00\x00\x00o\x00\x00\x00'
```

Could you please enable louis debug logging in the advanced settings panel and set logging to debug before reproducing? This should allow us to see what's going wrong in much more detail.

**Comment by DrSooom** [comment]
@Andre9642: Please try to reproduce it with [Try-18797](https://ci.appveyor.com/api/buildjobs/w66w8033f0mf3qcj/artifacts/output%2Fnvda_snapshot_try-initialDisplayFix-18797%2Ccf53fa2e.exe) and also read [my comment here in issue #9982](https://github.com/nvaccess/nvda/issues/9982#issuecomment-535513789). Maybe these issues are related.

Therefore please select your BAUM VarioUltra 40 manually from the Select Braille Display Dialog and save that configuration. In issue #9982 I didn't test the "Revert configuration" feature (NVDA+Ctrl+R) – I only took a look on NVDA's startup process.

**Comment by Andre9642** [comment]
@DrSooom I've just tested this try build and it's better. Braille no longer crashes.
I tried with and without auto-detection, same result. However during several tests, I received another error:

```
Core frozen in stack:
  File "nvda.pyw", line 215, in <module>
  File "core.pyc", line 532, in main
  File "wx\core.pyc", line 2134, in MainLoop
  File "gui\__init__.pyc", line 1000, in Notify
  File "core.pyc", line 502, in run
  File "queueHandler.pyc", line 83, in pumpAll
  File "queueHandler.pyc", line 50, in flushQueue
  File "speech\__init__.pyc", line 101, in cancelSpeech
  File "speech\manager.pyc", line 426, in cancel
  File "synthDrivers\espeak.pyc", line 146, in cancel
  File "synthDrivers\_espeak.pyc", line 241, in stop
  File "nvwave.pyc", line 311, in stop
  File "nvwave.pyc", line 283, in _idleUnbuffered
```

And NVDA crashes completely. I'll make more test soon.
Thanks.

**Comment by lukaszgo1** [comment]
@Andre9642 How it behaves with latest Alpha?

**Comment by Andre9642** [comment]
Fixed via #10498
